server/src/test/java/org/apache/uniffle/server/storage/LocalStorageManagerTest.java

```diff
@@ -209,21 +218,33 @@ public void testInitializeLocalStorage() {
 
     // case2: when candidates exist, it should initialize successfully.
     conf.set(
-        ShuffleServerConf.RSS_STORAGE_BASE_PATH, Arrays.asList("/a/rss-data", "/tmp/rss-data-1"));
+        ShuffleServerConf.RSS_STORAGE_BASE_PATH,
+        Arrays.asList(storageBaseDir1.getAbsolutePath(), rootRestrictedDir1.getAbsolutePath()));
     LocalStorageManager localStorageManager = new LocalStorageManager(conf);
     assertEquals(1, localStorageManager.getStorages().size());
 
     // case3: all ok
     conf.set(
         ShuffleServerConf.RSS_STORAGE_BASE_PATH,
-        Arrays.asList("/tmp/rss-data-1", "/tmp/rss-data-2"));
+        Arrays.asList(storageBaseDir1.getAbsolutePath(), storageBaseDir2.getAbsolutePath()));
     localStorageManager = new LocalStorageManager(conf);
     assertEquals(2, localStorageManager.getStorages().size());
 
-    // case4: only have 1 candidate, but exceed the number of
+    // case4: after https://github.com/apache/incubator-uniffle/pull/616
+    // dirs will be created automatically if they do not exist
```

Comment:
after https://github.com/apache/incubator-uniffle/pull/616, dirs will be created automatically if they do not exist. This means the test cases in `testInitializeLocalStorage` have been outdated for a long time.